deploy/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:3.5
+
+LABEL maintainer="adundovi@gmail.com"
+LABEL description="Prepares a docker image \
+of pump.io server, http://pump.io/."
+
+ENV LC_ALL en_US.UTF-8
+
+RUN apk add --no-cache \
+  nodejs \
+  git \
+  python \
+  make \
+  g++ \
+  graphicsmagick
+RUN npm install -g databank-mongodb
+RUN npm install -g pump.io
```

Comment:
I think it would be better to find a way to have this install from the source tree instead of from npm. What do you think?